concrete/src/Http/DefaultDispatcher.php

```diff
@@ -64,16 +64,14 @@ public function dispatch(SymfonyRequest $request)
     private function getEarlyDispatchResponse()
     {
         $validator = $this->app->make(SessionValidator::class);
-        if ($validator->hasActiveSession()) {
-            $session = $this->app['session'];
-            if (!$session->has('uID')) {
-                User::verifyAuthTypeCookie();
-            }
+        $session = $this->app['session'];
```

Comment:
I think that this line automatically initializes a new session if it's not yet started: we should call `$validator->hasActiveSession()` before it.

Reply:
Yeah, I agree – we can't look at session at all unless we've gone through $validator->hasActiveSession() because looking at it will start the session.

But I wonder – could we simply add another conditional to $validator->hasActiveSession() that returns true if the user has the ccmAuthUser cookie?